### DISCUSSION TITLE
2024/03/29/ir-tracking

### DISCUSSION THREAD

**Discussion by giscus** [NuclearPhoenixx/nuclearphoenixx.github.io#4] [Announcements]
# 2024/03/29/ir-tracking

I want to show you another project that I have been working on in the mean time. I needed good face tracking for my flight sim setup for which I was actually using Opentrack with AItrack previously (until I noticed that this feature was already built into Opentrack itself lol). However, face tracking wasn’t doing it for me anymore. I really don’t like the fact that you always have to be perfectly lit and you cannot tilt your head too much or the tracker will lose your face. It wasn’t the most responsive either, so I wanted to switch to IR tracking!

https://nuclearphoenix.xyz/2024/03/29/ir-tracking

**Comment by NuclearPhoenixx** [comment]
Hello World, this is my first comment!

**Reply by NuclearPhoenixx** [reply]
Does it still work?

**Reply by NuclearPhoenixx** [reply]
Hell yeah, it does! Nice.
